src/app/app.component.ts

```diff
@@ -242,7 +255,8 @@ export class AppComponent {
    *          if given, the total number of results to create pages
    *          for, otherwise defaults to the current result count
    */
-  private createPages(totalResults = this.resultCount) {
+  private createPages(totalResults = this.resultCount): void {
+    console.log(`Creating pages for ${totalResults} results`);
```

Comment:
Is the console.log necessary?

Reply:
Not any more - this dates back to debugging filters, when the total results would change if filtered.